test/experimental/test_transforms_with_asset.py

```diff
@@ -169,6 +181,17 @@ def test_builtin_pretrained_sentencepiece_processor(self):
         ref_results = [13, 1465, 12824, 304, 24935, 5771, 3776]
         self.assertEqual(spm_transform(test_sample), ref_results)
 
+    # we separate out these errors because Windows runs into seg faults when propagating
+    # exceptions from C++ using pybind11
+    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
```

Comment:
If we're disabling it for windows we can probably move the functions back into the text